lib/happy_hood/slack/client.rb

```diff
@@ -29,11 +29,23 @@ def self.daily_message
 
         def self.summarize_differences(differences)
           differences.reject { |d| d.valuation_difference.zero? }.map do |d|
-            "```"\
-            "(#{d.name}) - #{d.house_count} Happy Houses - (#{d.valuation_date})\n"\
```

Comment:
Hm since `valuation_date` is no longer used, I guess we can just delete it from the calculator too?

Reply:
Good catch!